TMDS_3/doc/TMDS_3/TMDS_3.txt

```diff
@@ -0,0 +1,91 @@
+namespace CGAL {
+/*!
+
+\mainpage User Manual
+\anchor Chapter_3D_Tetrahedral_Mesh_Data_Structure
+\anchor userchaptertmds3
+\authors Pierre Alliez, Clément Jamin, Laurent Rineau, Stéphane Tayeb, Jane Tournois, Mariette Yvinec
+\cgalAutoToc
+
+\cgalFigureBegin{figure_tetmesh_knot,knot_full.png}
+A multi-domain tetrahedral mesh generated from a polyhedral surface with multiple surface patches.
+The complete triangulation is displayed, including cells that belong or do not
+belong to the mesh complex, surface patches and feature edges.
+\cgalFigureEnd
+
+
+\section TMDS_3_section_c3t3 Mesh Complex
+
+This package is devoted to the representation of 3-Dimensional Tetrahedral Mesh Data Structures.
+
+A 3D linear cell complex is composed of points, line segments, polygons,
+polyhedral, and the combinatorial description of their n-dimension counterparts
+(namely vertices, edges, faces and cells).
+CGAL provides 3D triangulations, that describe
+both the geometry and connectivity of a 3D linear cell complex made
+of triangles, implemented in the
+the \ref PkgTriangulation3 and \ref PkgTDS3 packages.
```

Comment:
I still wonder why you define linear cell complexes. That seems off-topic. Why not define 3D simplicial complexes, instead of "3D linear cell complex made
of triangles"?